Answers.md

```diff
@@ -2,8 +2,13 @@
 
 1. What is React JS? How does it differ from other JavaScript frameworks?
 
+    ReactJS is a javascript library that controls DOM manipulation through its component system.
 
 2. Explain briefly the React Component Lifecycle. Name three of the methods that are a part of the lifecycle and what they do.
 
+    I would describe the react component lifecycle as  mount, update, and unmount. As I understand it, the first step is to create a component and initialize it with componentDidMount(). At this point, you can update it with the .setState() method. The last step would be to theoretically unmount the component, but I don't believe we've done this yet.

```

Comment:
Here are some good resources on component lifecycle [Link 1](https://reactarmory.com/resources/lifecycle-simulators) | [Link 2](https://engineering.musefind.com/react-lifecycle-methods-how-and-when-to-use-them-2111a1b692b1) | [Link 3](https://tylermcginnis.com/an-introduction-to-life-cycle-events-in-react-js/)